3rdparty/python/requirements.txt

```diff
@@ -6,6 +6,7 @@ freezegun==1.1.0
 # anonymity promise we make here: https://www.pantsbuild.org/docs/anonymous-telemetry
 humbug==0.2.6
 
+ijson==3.1.4
```

Comment:
You're using this rather than `json` for performance, to use iterators?

Reply:
No, I'm using it to be able to parse stdout which has multiple top-level JSON objects. The `json` module cannot decode that sort of document.